### PR TITLE
Add apirequestcounts.apiserver.openshift.io CRD

### PR DESCRIPTION
**Pull request by pradeepnaramaneni5**
ROKS 4.8: Add missing apirequestcounts.apiserver.openshift.io CRD #2359

Doc ref: https://github.com/openshift/enhancements/blob/master/enhancements/kube-apiserver/stability-api-removal-notifications.md

### CODE REVIEW

**Review by rtheis — CHANGES_REQUESTED** [review]
I believe this CRD belong with the cluster bootstrap assets since it must be applied on the cluster not the control plane.

**Comment by pradeepnaramaneni5** [comment]
@rtheis As suggested, updated back to cluster bootstrap assets folder.

File: assets/cluster-bootstrap/apiserver-apirequestcounts-crd.yaml [hunk 1]
```diff
@@ -0,0 +1,254 @@
+apiVersion: apiextensions.k8s.io/v1
```
Comment (rtheis) [inline assets/cluster-bootstrap/apiserver-apirequestcounts-crd.yaml:1]: https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.8/bindata/v4.1.0/kube-apiserver/apiserver.openshift.io_apirequestcount.yaml is the version used in OpenShift 4.8.  Please use this version.

**Review by rtheis — CHANGES_REQUESTED** [review]

File: assets/cluster-bootstrap/apiserver-apirequestcounts-crd.yaml (hunk 1 above)
Comment (pradeepnaramaneni5) [reply assets/cluster-bootstrap/apiserver-apirequestcounts-crd.yaml:1]: I was used the crd provided by Cesar from Our Chat. The one he had given was from Master branch. I have updated back the CRD to use 
https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.8/bindata/v4.1.0/kube-apiserver/apiserver.openshift.io_apirequestcount.yaml

**Review by rtheis — APPROVED** [review]
/lgtm

**Comment by rtheis** [comment]
@csrwng ptal

**Comment by rtheis** [comment]
I didn't intend to have this merge until Cesar approves.

**Pull request merged**